src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -27,6 +27,78 @@ public sealed partial class ApplicationIdentity : System.Runtime.Serialization.I
     }
 }
 
+namespace System.Data.Common
+{
+    [System.Security.Permissions.SecurityPermission(System.Security.Permissions.SecurityAction.InheritanceDemand, ControlEvidence = true, ControlPolicy = true)]
+    public abstract partial class DBDataPermission : System.Security.CodeAccessPermission, System.Security.Permissions.IUnrestrictedPermission
+    {
+        [System.Obsolete("DBDataPermission() has been deprecated.  Use the DBDataPermission(PermissionState.None) constructor.  http://go.microsoft.com/fwlink/?linkid=14202", true)]
```

Comment:
You don't need the string in [Obsolete] ... in fact, there's barely any value in having Obsolete since clearly everything in here is obsolete. If you want you can remove the strings from [Obsolete] throughout this file.